src/apps/components/AppPage/AppPageNav.tsx

```diff
@@ -0,0 +1,69 @@
+import { appsListUrl } from "@dashboard/apps/urls";
+import { TopNavLink, TopNavWrapper } from "@dashboard/components/AppLayout";
+import { LinkState } from "@dashboard/components/Link";
+import { AppAvatar } from "@dashboard/new-apps/components/AppAvatar/AppAvatar";
+import { GitHub, OfflineBoltOutlined } from "@material-ui/icons";
+import { Box, Button, Text } from "@saleor/macaw-ui/next";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import { useLocation } from "react-router";
+
+interface AppPageNavProps {
+  name: string | undefined;
+  supportUrl: string | undefined;
+  homepageUrl: string | undefined;
+}
+
+export const AppPageNav: React.FC<AppPageNavProps> = ({
+  name,
+  supportUrl,
+  homepageUrl,
+}) => {
+  const location = useLocation<LinkState>();
+  const goBackLink = location.state?.from ?? appsListUrl();
+
+  return (
+    <TopNavWrapper>
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent="space-between"
+        width="100%"
+      >
+        <Box display="flex" gap={7} alignItems="center">
+          <TopNavLink to={goBackLink} variant="tertiary" />
+          <Box display="flex" gap={5} alignItems="center">
+            <AppAvatar />
+            <Text variant="heading">{name}</Text>
+          </Box>
+        </Box>
+        <Box display="flex" gap={4}>
+          {supportUrl && (
+            <Button
+              variant="secondary"
+              size="medium"
+              onClick={() => {
+                window.open(supportUrl, "_blank");
+              }}
+            >
+              <GitHub />
```

Comment:
Im not sure if Github logo fits support page

Reply:
I've talked with @pflis1 and I've removed icons from those buttons :)